src/meta/server_state.h

```diff
@@ -171,6 +171,9 @@ class server_state
 
     void on_query_restore_status(configuration_query_restore_rpc rpc);
 
+    // manual compaction
```

Comment:
I think you can add some meaningful notes here

Reply:
The comment here is just like division, showing the following functions are about manual compaction, because `server_state.h` defined lots of functions. You can reference `server_state.h` to see similar usage, such as `app options`, `dump & restore`.